californium-core/src/test/java/org/eclipse/californium/core/test/lockstep/BlockwiseServerSideTest.java

```diff
@@ -798,6 +799,8 @@ public void testObserveWithBlockwiseResponse() throws Exception {
 
 		client.sendRequest(CON, GET, tok3, ++mid).path(RESOURCE_PATH).block2(2, false, 128).go();
 		client.expectResponse(ACK, CONTENT, tok3, mid).block2(2, false, 128).noOption(OBSERVE).payload(respPayload.substring(256, 290)).go();
+
+		testResource.clearObserveRelations();
```

Comment:
I'm not sure, what the intent of the "clear" is.
`testResource` is "destroyed" anyway with the `server.destroy()` in `shutdownEndpoints()`. 
Or why is this required?

Reply:
If we don't clean observe relations the exchangeStore is not empty. (as the observe relation is still established)
Another solution could be to make the client stop the observation. (but I choose this solution as the cancel seems to not be part of the test)